src/main/java/org/simlar/simlarserver/database/repositories/SubscriberRepository.java

```diff
@@ -34,4 +35,7 @@ public interface SubscriberRepository extends CrudRepository<Subscriber, Long> {
 
     @Query("SELECT id FROM Subscriber WHERE username = ?1 AND domain = ?2")
     List<Long> findIdByUsernameAndDomain(final String username, final String domain);
+
+    @Query("SELECT username FROM Subscriber WHERE domain = ?1 AND username IN ?2")
```

Comment:
> Be careful when passing in many values. The JDBC standard does not guarantee that you can use more than 100 values for an IN expression list. Various databases exceed this number, but they usually have a hard limit for how many values are allowed. For example, Oracle’s limit is 1000.

https://docs.spring.io/spring-framework/reference/data-access/jdbc/parameter-handling.html#jdbc-in-clause